src/Pos/Update/Logic/Local.hs

```diff
@@ -179,7 +179,7 @@ processSkeleton
 processSkeleton payload = withUSLock $ runExceptT $ withCurrentTip $ \ms@MemState{..} -> do
     modifier <-
         runDBPoll . evalPollT msModifier . execPollT def $
-        verifyAndApplyUSPayload (Left msSlot) payload
+        verifyAndApplyUSPayload False (Left msSlot) payload
```

Comment:
seems, `True` must be here, because we accept all proposals (even bad for us), and if we try to create a block we'll fail.